### PR TITLE
Pause / Unpaused Report Format

### LINKED ISSUE #2686: Update Polls Reports for Suspension
See the [Tech Spec](https://docs.google.com/document/d/15mSk4QPQUqsLlHFwF5Dw058OsprmoJixwxm7C2GmGGk/edit?usp=sharing) for the overall https://github.com/votingworks/vxsuite-roadmap/issues/37.

Update the polls open report to:
- Include titles and metadata relevant to polls suspension
- Have different report format for polls suspension and unsuspension, which only includes ballot counts in addition to metadata

### PR DESCRIPTION
**Pull request by adghayes**
## Overview
Closes #2686. Creates a new report format for polls paused and unpaused which only includes the total ballot count, and not the results.

Has a couple of commits on the front making some agreed upon changes - using "reopen" instead of "open" for unpausing, and removing the precinct name from the polls action buttons.

## Demo Video or Screenshot

paused_rev3

reopened_rev3

## Checklist
- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/utils/src/polls.ts [hunk 1]
```diff
@@ -70,7 +72,7 @@ export function getPollTransitionsFromState(
     case 'polls_open':
       return ['close_polls', 'pause_polls'];
     case 'polls_paused':
-      return ['open_polls', 'close_polls'];
+      return ['unpause_polls', 'close_polls'];
```
Comment (adghayes) [inline libs/utils/src/polls.ts:75]: Making the language explicit in the UI helped identify this actual error :)

Comment (jonahkagan) [reply libs/utils/src/polls.ts:75]: Is there a test that covers this?

File: libs/ui/src/precinct_scanner_ballot_count_report.test.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      isLiveMode={false}
+      precinctScannerMachineId="SC-01-000"
+    />
+  );
+
+  // Check header
+  screen.getByText('TEST Polls Paused Report for All Precincts');
+  const electionTitle = screen.getByText('General Election:');
+  expect(electionTitle.parentElement).toHaveTextContent(
+    'General Election: Tuesday, November 3, 2020, Franklin County, State of Hamilton'
```
Comment (jonahkagan) [inline libs/ui/src/precinct_scanner_ballot_count_report.test.tsx:29]: It seems weird to me to have the election name styled as a label for the rest of the election details ("<Election name>: <date, count, state>"), while the other things in the report styled that way are actually labels (e.g. "Polls Paused" and "Scanner ID")

File: libs/ui/src/precinct_scanner_tally_qrcode.tsx [hunk 3]
```diff
@@ -73,8 +70,8 @@ export function PrecinctScannerTallyQrCode({
           {electionDate}, {election.county.name}, {election.state}
           <br />
           <Text small as="span">
-            Polls {pollsAction} and report created on{' '}
-            {formatFullDateTimeZone(DateTime.fromMillis(pollsToggledTime))}
+            Polls Closed and report created on{' '}
```
Comment (jonahkagan) [inline libs/ui/src/precinct_scanner_tally_qrcode.tsx:73]: I might lowercase the action here to make it read more like a sentence

**Review by jonahkagan — APPROVED** [review]

File: libs/utils/src/polls.ts (hunk 1 above)
Comment (adghayes) [reply libs/utils/src/polls.ts:75]: https://github.com/votingworks/vxsuite/pull/2764/commits/63cc1eddb097e56a4bd22babcfe617ec2b13418e#diff-797af4cc4e0b6e4d7b559398a2027267f42f8828ca9f5afd0dae710e73c61315R104

Comment (adghayes) [reply libs/utils/src/polls.ts:75]: These sorts of tests where the assertions just basically rehash the behavior of the utility seem useless to me.

File: libs/ui/src/precinct_scanner_tally_qrcode.tsx (hunk 3 above)
Comment (adghayes) [reply libs/ui/src/precinct_scanner_tally_qrcode.tsx:73]: Good catch.

Comment (adghayes) [reply libs/ui/src/precinct_scanner_tally_qrcode.tsx]: Fixed

File: libs/ui/src/precinct_scanner_ballot_count_report.test.tsx (hunk 2 above)
Comment (adghayes) [reply libs/ui/src/precinct_scanner_ballot_count_report.test.tsx:29]: I'll leave this as is, but noting the feedback for @beausmith.

File: libs/utils/src/polls.ts (hunk 1 above)
Comment (adghayes) [reply libs/utils/src/polls.ts:75]: But now that the language is distinct, this is covered in a few places in the frontends.

**Pull request merged**